Automattic/jetpack
# My plan: update feature cards

Fixes #10582

#### Changes proposed in this Pull Request:

* Updates plan feature cards content.
* Updates plan feature cards design.

#### Before

![image](https://user-images.githubusercontent.com/390760/49738571-94152b80-fc87-11e8-997f-f4e3cef67915.png)

#### After

![image](https://user-images.githubusercontent.com/390760/49738877-6381c180-fc88-11e8-99d3-83441d614359.png)



#### Testing instructions:

* Fire up this branch on your site.
* To to `/wp-admin/admin.php?page=jetpack#/my-plan`.
* Make sure you see the feature cards with updated content and styles.

#### Proposed changelog entry for your changes:


* My plan: update content and design of plan feature cards.
